modules/virt-pausing-vm-web.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
```

Comment:
New file copied from [virt-starting-vm-web.adoc](https://github.com/openshift/openshift-docs/pull/69051/files#diff-8cb9d109052efd28a6ee8492466998313d6ed6fd2138f268473f8d79c924d581) and changed _start_ to _pause_.